mcp/worker.ts

```
@@ -0,0 +1,1481 @@
+import { Honcho } from '@honcho-ai/sdk';
+
+interface Env {
+    // You can add environment variables here if needed
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)